Sources/SwiftSyntaxBuilderGeneration/gyb_generated/AvailabilityNodes.swift

```diff
@@ -0,0 +1,103 @@
+//// Automatically Generated From AvailabilityNodes.swift.gyb.
+//// Do Not Edit Directly!
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+let AVAILABILITY_NODES: [Node] = [
+  Node(name: "AvailabilitySpecList",
+       kind: "SyntaxCollection",
+       element: "AvailabilityArgument"),
+
+  Node(name: "AvailabilityArgument",
+       description: "A single argument to an `@available` argument like `*`, `iOS 10.1`,or `message: \"This has been deprecated\"`.",
+       kind: "Syntax",
+       children: [
+        Child(name: "Entry",
```

Comment:
This is currently indented by one space, it should probably be two.

Reply:
I used the indentation from Xcode, and afterwards adjusted the `gyb` to it.

It might be something in Xcode then 🤷‍♂️

Will change. 
Should the wrong indent bug be reported ?

Reply:
If you have a reproducer in Xcode, it would be great if you could file a bug report on http://feedbackassistant.apple.com.

Reply:
Openend: FB10505542

Reply:
Thanks for filing the bug report 👍